Apps/WebClient/src/ClientApp/app/components/navmenu/navHeader.vue

```diff
@@ -130,9 +112,7 @@ export default class HeaderComponent extends Vue {
     }
   }
 
-  created() {
-    this.loadLanguages();
-  }
+  
 
   get displayMenu(): boolean {
```

Comment:
This method doesnt look necessary anymore

Reply:
okay, will remove